openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -406,7 +406,7 @@ private void generatesClientFile(BallerinaCodeGenerator generator, Path resource
                                      boolean resourceMode, boolean statusCodeBinding) {
         try {
             generator.generateClient(resourcePath.toString(), targetOutputPath.toString(), filter, baseCmd.nullable,
-                    resourceMode, statusCodeBinding, baseCmd.mock);
+                    resourceMode, statusCodeBinding, baseCmd.mock, baseCmd.singleFile);
```

Comment:
Since this option can be used only with either client or service generation, shall we print a warning when this is used for both generation saying that single file is not supported and defaulting to generate separate files